avocado/core/utils.py

```diff
@@ -80,5 +79,5 @@ def resolutions_to_tasks(resolutions, config):
                                                  include_empty,
                                                  include_empty_key):
                     continue
-            tasks.append(Task(str(uuid1()), runnable, [status_server]))
+            tasks.append(Task(runnable, None, [status_server]))
```

Comment:
There is no need to have `None` here.

Reply:
Are you sure? The parameters are being passed based on their position.  If this parameter is omitted, then `[status_server]` is taken as an identifier.

Reply:
True, sorry that I didn't describe it correctly. I mean that we should use `Task(runnable,  status_uris=[status_server])`

Reply:
But that's just my preference